init.sql

```diff
@@ -16,3 +16,19 @@ CREATE TABLE IF NOT EXISTS CourseEnrollments(
     FOREIGN KEY (student_id) REFERENCES Students(id),
     FOREIGN KEY (course_id) REFERENCES Courses(id)
 );
+
+CREATE TABLE IF NOT EXISTS Tutors(
```

Comment:
queste create mettile in un file a parte sotto la tua cartella personale, non modificare l'init